.github/actions/setup-env/action.yml

```
@@ -47,3 +52,8 @@ runs:
         elif [ -n "$NOX_SETUP" ]; then
           nox -s setup -- $NOX_SETUP
         fi
+    - name: Set python version
+      id: python-version
+      shell: bash
+      run: |
+        echo '::set-output name=python-version::' $(python -c 'import sys; print(".".join(map(str,sys.version_info[:2])))')
```

```suggestion
        echo "::set-output name=python-version::$(python -c 'import sys; print(".".join(map(str,sys.version_info[:2])))')"
```
small nit, avoids extra space in output value ("3.11" instead of " 3.11")

other than that, lgtm